codegen/java-gen/src/test/resources/tests/action-service/simple-old-style/generated-managed/org/example/AkkaServerlessFactory.java

```diff
@@ -0,0 +1,23 @@
+package org.example;
+
+import com.akkaserverless.javasdk.AkkaServerless;
+import com.akkaserverless.javasdk.action.ActionCreationContext;
+import org.example.service.MyServiceAction;
+import org.example.service.MyServiceActionProvider;
+import org.example.service.ServiceOuterClass;
+
+import java.util.function.Function;
+
+// This code is managed by Akka Serverless tooling.
+// It will be re-generated to reflect any changes to your protobuf definitions.
+// DO NOT EDIT
+
+public final class AkkaServerlessFactory {
+
+  public static AkkaServerless withComponents(
+      Function<ActionCreationContext, MyServiceAction> createMyServiceAction) {
+    AkkaServerless akkaServerless = new AkkaServerless();
+    return akkaServerless
+      .register(MyServiceActionProvider.of(createMyServiceAction));
+  }
+}
```

Comment:
These managed/generated ones, should they really be checked in, vs regenerated on build?

Reply:
That's the goal. It allow us to do first check that what we generate is indeed what we want (now we can even compile them) and they help us to see how a change in the codegen will affect the user. 

Now, we have different tests styles in the codegen and we should start to clean it up. We probably have a few that are becoming irrelevant, but I will write an issue about it later.